gh2/csv.py

```diff
@@ -174,12 +177,17 @@ def normalize_sequential_dates(issue_list):
 
     return issue_list
 
+def write_headers(filename, headers):
+    with open(filename, 'w+') as fd:
+        writer = csv.writer(fd)
+        writer.writerow(headers)
+        fd.close()
```

Comment:
yup, for some reason it was not behaving properly when I first tested it, but you're right.